Tevling/Components/AthleteCard.razor.css

```diff
@@ -23,7 +23,11 @@
 
     .unfollow-button,
     .follow-button {
-        max-width: 6rem;
+        max-width: fit-content;
+    }
+    
+    .unfollow-button-text, .follow-button-text {
+        display: none;
```

Comment:
I think you might have to override `.unfollow-button:hover::after` here as well. At least when I tested in the browser it looked a bit glitchy with the "Unfollow" text showing up for a short moment.